src/providers/bedrock/complete.ts

```diff
@@ -125,6 +125,37 @@ export const BedrockLLamaCompleteConfig: ProviderConfig = {
   },
 };
 
+export const BedrockMistralCompleteConfig: ProviderConfig = {
+  prompt: {
+    param: 'prompt',
+    required: true,
+  },
+  max_tokens: {
+    param: 'max_tokens',
+    default: 20,
+    min: 1,
+  },
+  temperature: {
+    param: 'temperature',
+    default: 0.75,
+    min: 0,
+    max: 5,
+  },
+  top_p: {
+    param: 'top_p',
+    default: 0.75,
+    min: 0,
+    max: 1,
+  },
+  top_k: {
+    param: 'top_k',
+    default: 0,
+  },
+  stop: {
```

Comment:
As mentioned in this comment, stop can be mapped to stop itself.
https://github.com/Portkey-AI/gateway/pull/268/files#r1544562206